app/models/financialDetails/DocumentDetail.scala

```diff
@@ -45,6 +45,10 @@ case class DocumentDetail(taxYear: Int,
                           documentDueDate: Option[LocalDate] = None
                          ) {
 
+  def isPoAOne: Boolean = documentDescription.exists(_.equals("ITSA- POA 1"))
```

Comment:
this would make sense to move these to methods to ClaimToAdjustService, as this is the only place where these are used